packages/components/pie-modal/src/defs.ts

```diff
@@ -161,7 +162,6 @@ export const defaultProps: DefaultProps = {
     isDismissible: false,
     isFooterPinned: true,
     isFullWidthBelowMid: false,
-    isLoading: false,
```

Comment:
Just wondering why you wanted/needed to remove this, `false` seems like a sensible default value to me

Reply:
because there's no reason to announce it is NOT loading if it never is expected to load. 

If it loads: announce its busy
Loading finishes: its not busy

Never was loading: why mention it at all?